### ISSUE TITLE
cannot import name 'OT' from 'OT'

### ISSUE THREAD

**Issue by rilakely** [NivKonst/Actively-Secure-Vector-OLE#1]
Traceback (most recent call last):
  File "Vector_OLE.py", line 14, in <module>
    from OT import OT
ImportError: cannot import name 'OT' from 'OT' (unknown location)

**Comment by NivKonst** [comment]
Hi.
Did you put the OT.pyd file in the OT directory?
If not, you can use a simple "Import OT".

**Comment by rilakely** [comment]
Hi, Thanks for your response. Yes I put OT.pyd file in the OT directory however it looks like .pyd is built in Windows environment I can't use it as I used Linux.

**Comment by NivKonst** [comment]
Ok, that may be the problem.
Note that you can use any Oblivious Transfer algorithm as a python file for our implementation.
Just make sure that it has the correct methods and parameters as stated in the Vector_OLE file.

**Comment by rilakely** [comment]
Yes. I think so. Can you provide any Oblivious Transfer algorithm? Based on your codebase, as I can't access to your OT, I have to implement OT.ActiveReceiver and OT.ActiveReceiver functions. 
Can you provide your original code about those? Thank you

**Comment by NivKonst** [comment]
We used the libOTe implementation:
https://github.com/osu-crypto/libOTe

It's a C++ implementation, and we used C++ Extensions for Python in order to use it and create a pyd file.
